Automatic Validation ended with: 
> 2026-02-15T19:42:01.3872968Z ##[error] Manifest Error: Schema header not found. File: tmtrckr.TimeTracker.installer.yaml Manifest Error: Schema header not found. File: tmtrckr.TimeTracker.locale.en-US.yaml Manifest Error: Schema header not found. File: tmtrckr.TimeTracker.yaml

(Automated response - build 1221.)

--- a/manifests/t/tmtrckr/TimeTracker/0.4.1/tmtrckr.TimeTracker.installer.yaml
+++ b/manifests/t/tmtrckr/TimeTracker/0.4.1/tmtrckr.TimeTracker.installer.yaml
@@ -1,4 +1,4 @@
-# Created with GitHub Actions
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.10.0.schema.json
 PackageIdentifier: tmtrckr.TimeTracker
 PackageVersion: 0.4.1
 Installers:

--- a/manifests/t/tmtrckr/TimeTracker/0.4.1/tmtrckr.TimeTracker.installer.yaml
+++ b/manifests/t/tmtrckr/TimeTracker/0.4.1/tmtrckr.TimeTracker.installer.yaml
@@ -1,4 +1,4 @@
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.10.0.schema.json
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.10.0.schema.json
 PackageIdentifier: tmtrckr.TimeTracker
 PackageVersion: 0.4.1
 Installers:

--- a/manifests/t/tmtrckr/TimeTracker/0.4.1/tmtrckr.TimeTracker.locale.en-US.yaml
+++ b/manifests/t/tmtrckr/TimeTracker/0.4.1/tmtrckr.TimeTracker.locale.en-US.yaml
@@ -1,4 +1,4 @@
-# Created with GitHub Actions
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.10.0.schema.json
 PackageIdentifier: tmtrckr.TimeTracker
 PackageVersion: 0.4.1
 PackageLocale: en-US

